sdk/cosmos/azure-cosmos-examples/src/main/java/com/azure/cosmos/rx/examples/multimaster/Helpers.java

```diff
@@ -38,7 +38,7 @@ static public String createDatabaseUri(String databaseName) {
                                 }
                             }
 
-                            return Flux.error(e);
+                            return Mono.error(e);
                         }
                 ).map(ResourceResponse::getResource).single();
```

Comment:
We don't need this `.single()` anymore, right ?
Here and every other method in this class.

Reply:
as discussed offline. they are intentionally kept as they will do correctness validation to ensure Mono is not empty.